docs/docs/self-host/customize-deployment/use-sso-login-for-self-hosted-lightdash.mdx

```diff
@@ -2,20 +2,93 @@
+
+- **Login initiated by**: App and Okta Sign-in Page
+- **Application visibility**: Display application icon to users
+- **Login flow**: Redirect to app to initiate login (OIDC Compliant)
+- **Initiate login URI**: `{{ lightdash_url }}/api/v1/login/okta`
+
+Hit **Save** to finish.
+
+### Okta configuration variables
+
+From the application settings page, you'll need to copy the following values:
+
+- **Client ID**
+- **Client secret**
+
+You'll also need your **Okta domain**, which is the first part of your okta URL. For example if your Okta URL is
+https://dev-123456.okta.com then your Okta domain is `dev-123456.okta.com`.
+
+Finally, you need the **Issuer URI**. This is the URL of your Okta authorization server. You can use
+your [Org authorization server](https://developer.okta.com/docs/concepts/auth-servers/#org-authorization-server)
+which uses `https://dev-123456.okta.com` as your issuer or select a custom authorization server. To find the issuer URI
+for a custom authorization server navigate to **API** > **Authorization Servers** and click on the authorization
+server and note the **Issuer URI** and **Name** of the authorization server. For example the `default` authorization
+server has an issuer URI of `https://dev-123456.okta.com/oauth2/default`.
+
+### Configuring Lightdash for Okta
+
+You'll need to set the following environment variables in your Lightdash deployment:
+
+| Variable                            | Description                                                                                        | Required? |
+|-------------------------------------|----------------------------------------------------------------------------------------------------|-----------|
+| `AUTH_OKTA_DOMAIN`                  | The `{{ okta_domain }}`. Should not include `https://`                                             | ✅         |
+| `AUTH_OKTA_OAUTH_CLIENT_ID`         | The **Client ID** copied from the application settings in okta                                     | ✅         |
+| `AUTH_OKTA_OAUTH_CLIENT_SECRET`     | The **Client secret** copied from the application settings in okta                                 | ✅         |
+| `AUTH_OKTA_OAUTH_ISSUER`            | The **Issuer URI** copied from the authorization server. Should include `https://`                 | ✅         |
+| `AUTH_OKTA_AUTHORIZATION_SERVER_ID` | Optional. The **Name** of a custom authorization server if not using the org authorization server. |           |
+
+## Google
+
+These variables enable you to control Single Sign On (SSO) functionality for Google
+
+| Variable                               | Description                                          | Required? | Default |
+|----------------------------------------|------------------------------------------------------|-----------|---------|
+| `AUTH_GOOGLE_OAUTH2_CLIENT_ID`         | Required for Google SSO                              |           |         |
```

Comment:
Any reason the okta vars have required and google/one login aren't?

Reply:
Fixed ✅